knowledge-base/common-kb-stackoverflowexception-editing-circular-references.md

```diff
@@ -0,0 +1,63 @@
+---
+title: How to Resolve StackOverflowException When Editing Items with Circular References in ListView for Blazor
```

Comment:
Both the title and the page title adhere to the guidelines for how-to KBs. ([source](https://docs.telerik.com/style-guide/kb-articles#titles-headings-and-subheadings))

```suggestion
title: StackOverflowException When Editing Items with Circular References in ListView for Blazor
```